readers/api/requests.go

```diff
@@ -20,17 +20,13 @@ type listMessagesReq struct {
 	chanID   string
 	token    string
 	key      string
-	domainID string
```

Comment:
We should have domainID for readers too. 
I will create new ticket for adding domain_id to readers URL

Reply:
Yes, we should. @felixgateru

Reply:
@felixgateru Let's use a different PR for that, I'll merge this one.